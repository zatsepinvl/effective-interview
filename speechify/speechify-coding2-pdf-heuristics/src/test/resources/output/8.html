            <html><body>
            <div style="display: flex; flex-direction: row">
            <div style="width: 50%">
            <h1>Expected</h1>
            <hr />
            <p>In August of 1861, Dent took up Lots 13 and 14 on the western bank of the creek (see Figure 4.3) which like the other blocks on the creek, had never been claimed by their buyers after the auction sale449. He very quickly established two sawpits on the waterfront, and before the year was out, had sent 35 logs and 510 slabs of timber to Sydney in October 1861, just two months after first arriving on the creek450.</p>
<p>Slabs were flitches of timber obtained by splitting sections of logs cut to length, used in the construction of the settlers’ slab and bark huts, as shown in the photo below:</p>
<p>Figure 4.5: Example of a Settler s cottage with slab walls and bark roofing. Source: Gerard Charles Wilson, gerardcharleswilson.com.</p>
<p>The seventeen ton ketch Sarah Dent, bought only in 1861, was sold and replaced in 1862 by a fifty ton schooner, the Sylvanus451. By 1864 he and his sons had built another ship, the Young Australian, a three masted schooner of 131 tons452, under the direction of a shipwright. Dent sent his new ship to Sydney with a load of timber, described as “the finest cargo of hardwood ever seen in this port”453. The hardwood was sold for shipbuilding purposes and was well received: “some of the pieces are 70 feet long and</p>
<p><i>449 New South Wales State Archives, Joseph Alexander Dent Primary Application 19667 Series 17513: statutory declarations by George Thomson, John Wilson and Christopher Joseph Murray. 450 See Table 4.1. 451 Both the Sarah Dent and the Sylvanus were built by Thomas Davis at Brisbane Water, and George Dent sold the Sarah Dent to Thomas Davis’s brother, Rock, another shipwright of Brisbane Water. Gwen Dundon, Shipbuilders of Brisbane Water (PO Box 202 East Gosford: Gwen Dundon, 1997), 131-132. 452 Argus (Melbourne); 15 June 1877, 7. 453 Empire, 2 April 1864, 5.</i></p>
<p><i>111</i></p>
            </div>
            <div style="width: 50%">
            <h1>Actual</h1>
            <hr />
            
            </div>
            </div>
            </body></html>