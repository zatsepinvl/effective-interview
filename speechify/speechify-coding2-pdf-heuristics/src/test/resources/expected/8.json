[
  {
    "type": "Paragraph",
    "text": "In August of 1861, Dent took up Lots 13 and 14 on the western bank of the creek (see Figure 4.3) which like the other blocks on the creek, had never been claimed by their buyers after the auction sale449. He very quickly established two sawpits on the waterfront, and before the year was out, had sent 35 logs and 510 slabs of timber to Sydney in October 1861, just two months after first arriving on the creek450."
  },
  {
    "type": "Paragraph",
    "text": "Slabs were flitches of timber obtained by splitting sections of logs cut to length, used in the construction of the settlers\u2019 slab and bark huts, as shown in the photo below:"
  },
  {
    "type": "Paragraph",
    "text": "Figure 4.5: Example of a Settler s cottage with slab walls and bark roofing. Source: Gerard Charles Wilson, gerardcharleswilson.com."
  },
  {
    "type": "Paragraph",
    "text": "The seventeen ton ketch Sarah Dent, bought only in 1861, was sold and replaced in 1862 by a fifty ton schooner, the Sylvanus451. By 1864 he and his sons had built another ship, the Young Australian, a three masted schooner of 131 tons452, under the direction of a shipwright. Dent sent his new ship to Sydney with a load of timber, described as \u201cthe finest cargo of hardwood ever seen in this port\u201d453. The hardwood was sold for shipbuilding purposes and was well received: \u201csome of the pieces are 70 feet long and"
  },
  {
    "type": "Marginalia",
    "text": "449 New South Wales State Archives, Joseph Alexander Dent Primary Application 19667 Series 17513: statutory declarations by George Thomson, John Wilson and Christopher Joseph Murray. 450 See Table 4.1. 451 Both the Sarah Dent and the Sylvanus were built by Thomas Davis at Brisbane Water, and George Dent sold the Sarah Dent to Thomas Davis\u2019s brother, Rock, another shipwright of Brisbane Water. Gwen Dundon, Shipbuilders of Brisbane Water (PO Box 202 East Gosford: Gwen Dundon, 1997), 131-132. 452 Argus (Melbourne); 15 June 1877, 7. 453 Empire, 2 April 1864, 5."
  },
  { "type": "Marginalia", "text": "111" }
]

